composer.json

```diff
@@ -13,11 +13,11 @@
     "minimum-stability": "stable",
     "require": {
         "php": ">=7.0.0",
-        "guzzlehttp/guzzle": "^6.3",
-        "illuminate/support": "~5.0|~6.0|~7.0"
+        "guzzlehttp/guzzle": ">=6.3",
```

Comment:
It's still better to define this as `^6.3|^7.0` against the specific versions as otherwise it would automatically accept Guzzle 8.0 if it ever is released and could end up breaking this version.